src/generate-pdf/generate-pdf.controller.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  Headers,
+  HttpStatus,
+  InternalServerErrorException,
+  Post,
+  Res,
+} from '@nestjs/common';
+import { ApiBody, ApiTags } from '@nestjs/swagger';
+import { correlationId, LoggingService } from '@s3pweb/nestjs-common';
+import { validateOrReject } from 'class-validator';
+import { Response } from 'express';
+import { GeneratePdfFromHtmlDto } from './dto/generate-pdf-html-dto';
+import { GeneratePdfFromUrlDto } from './dto/generate-pdf-url-dto';
+import { GeneratePdfService } from './generate-pdf.service';
+
+@ApiTags('generate')
+@Controller('v2/generate-pdf')
+export class GeneratePdfController {
+  constructor(
+    private readonly generatePdfService: GeneratePdfService,
+    private readonly logger: LoggingService,
+  ) {}
+
+  @Post('url')
+  @ApiBody({ type: GeneratePdfFromUrlDto })
+  async generateFromUrlParams(
+    @Body()
+    body: GeneratePdfFromUrlDto,
+    @Res() res: Response,
+    @Headers(correlationId) uuid: string,
+  ): Promise<void> {
+    const { url, format, waitFor } = body;
```

Comment:
@steve3524 Je te laisse ajouter un peu plus de logs sur tes routes et différentes fonctions (petit exemple ci-dessous) 

this.log.info({ uuid }, `Assist order created: ${JSON.stringify(assistOrder)}`);